client/lib/abtest/README.md

```diff
@@ -1,3 +1,9 @@
+
+# This library is deprecated!
+
+*Please consider ExPlat for your next A/B experiment.*
+See pbxNRc-sZ-p2 or visit #a8c-experiments and ask around!
```

Comment:
It's probably better to link to `/client/components/experiment/readme.md` and add the P2 and Slack links there.